src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLogger.cs

```diff
@@ -205,20 +203,13 @@ private void ModernPath(LogLevel logLevel, EventId eventId, LoggerMessageState m
                     jr.Next = jitRedactors;
                     jitRedactors = jr;
 
-                    var index = msgState.ReserveRedactedTagSpace(1);
-                    msgState.RedactedTagArray[index] = new(cp.Name, jr);
+                    msgState.RedactedTagArray[i] = new(cp.Name, jr);
                 }
                 else
                 {
-                    var index = msgState.ReserveRedactedTagSpace(1);
-                    msgState.RedactedTagArray[index] = new(cp.Name, null);
+                    msgState.RedactedTagArray[i] = new(cp.Name, null);
                 }
             }
-            catch (Exception ex)
```

Comment:
Does that mean that we can;t throw here anymore?

Reply:
Correct, this code is no longer calling outside of this module and so isn't vulnerable to exceptions.